### PR TITLE
Fixes #3606

### LINKED ISSUE #3606: 1.8.1版本未开启auth并且配置了context-path后访问portal后台页面，部分资源加载404
- [x] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [x] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [x] 我已经仔细检查过[FAQ](https://ctripcorp.github.io/apollo/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**

基于1.8.1版本
portal server 配置apollo_profile未启用auth，并且配置了context-path后访问portal后台页面，部分资源加载404（404请求未带上context-path所配置的路径前缀）

**复现**

通过如下步骤可以复现：
1. portal server配置context-path
2. portal server不使用auth启动
3. 访问portal server

**期望**

所有资源正常访问

**问题原因**

portal不带auth启动时访问页面为index.html，但在该html中未先请求获取prefixPath，导致后续请求资源路径错误

**截图**

1615620227293_D3A1D438-D49F-4DBA-B485-334B3339A627

### 额外的细节和日志

- 版本：1.8.1
- 错误日志
- 配置：
- 平台和操作系统

**修改参考**
1. portal中app.html的121行引用路径问题，修改为
    <script src="vendor/lodash.min.js"></script>

2. portal中index.html的125行到127行中间（该需在加载app.js之前）加入获取context-path请求（可以先判断本地是否存在prefixPath然后在请求）
    <script type="application/javascript">
        prefix = function (){
            $.ajax({
                method: 'get',
                async: false,
                url: 'prefix-path',
                success: function (res) {
                    window.localStorage.setItem("prefixPath", res);
                }
            })
        };
        prefix();
    </script>

### PR DESCRIPTION
**Pull request by Loading-Life**
## What's the purpose of this PR

用来解决portal未启用auth并且配置了context-path后导致的, 访问首页部分资源加载404问题

## Which issue(s) this PR fixes:
Fixes #3606

## Brief changelog

1. portal - app.html 中 lodash.min.js 引用路径修改
2. index.html 中 加入获取prefixPath逻辑

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.

### CODE REVIEW

File: apollo-portal/src/main/resources/static/index.html [hunk 1]
```diff
@@ -124,6 +124,20 @@ <h5 ng-bind="app.name"></h5>
     <!-- bootstrap.js -->
     <script src="vendor/bootstrap/js/bootstrap.min.js" type="text/javascript"></script>
 
+    <script type="application/javascript">
```
Comment (nobodyiam) [inline apollo-portal/src/main/resources/static/index.html:127]: How about we use the same code as that in [login.html](https://github.com/ctripcorp/apollo/blob/master/apollo-portal/src/main/resources/static/login.html)?

```javascript
    <script type="application/javascript">
        $(function () {
            $.get("prefix-path", function (result) {
                window.localStorage.setItem("prefixPath", result);
            });
        });
    </script>
```

Comment (Loading-Life) [reply apollo-portal/src/main/resources/static/index.html:127]: 异步会有加载先后顺序问题，导致prefix-path还未获取，就已经开始加载需要前缀的资源了，从而引起了404问题

Comment (nobodyiam) [reply apollo-portal/src/main/resources/static/index.html:127]: 那[login.html](https://github.com/ctripcorp/apollo/blob/master/apollo-portal/src/main/resources/static/login.html)中的这段代码是否也要做同样的修改?

```javascript
    <script type="application/javascript">
        $(function () {
            $.get("prefix-path", function (result) {
                window.localStorage.setItem("prefixPath", result);
            });
        });
    </script>
```

Comment (Loading-Life) [reply apollo-portal/src/main/resources/static/index.html:127]: 测试了一下，的确也有上述异步加载问题（会导致zh-CN.json文件404）。又补充提交了 login.html 的修改

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**